huxley/core/middlewares.py

```diff
@@ -12,3 +12,18 @@ class ServerLoggingMiddleware(object):
     def process_exception(self, request, exception):
         logger = logging.getLogger('huxley.server')
         logger.exception(exception)
+
+class LoggingMiddleware(object):
+    def process_response(self, request, response):
+        if 'api' in request.path:
+            logger = logging.getLogger('huxley.api')
+            uri = request.path
```

Comment:
Why is this assigned to a local variable? `uri`, `message` and `status_code` can be inlined into the dictionary.